lms/djangoapps/certificates/tests/tests.py

```diff
@@ -58,7 +59,7 @@ def test_certificate_status_for_student(self):
     @data(
         {'allow_certificate': False, 'whitelisted': False, 'grade': None, 'output': ['N', 'N', 'N/A']},
         {'allow_certificate': True, 'whitelisted': True, 'grade': None, 'output': ['Y', 'N', 'N/A']},
-        {'allow_certificate': True, 'whitelisted': False, 'grade': 0.9, 'output': ['Y', 'N', 'N/A']},
+        {'allow_certificate': True, 'whitelisted': False, 'grade': 0.9, 'output': ['N', 'N', 'N/A']},
```

Comment:
Why is this changed? Is this user is in audit/honor mode in this test? Can we test its enrollment to be clear?